06-building-a-program-start-to-finish/setting-up-linters/README.md

```diff
@@ -0,0 +1,11 @@
+#### Output
+- **Using**: Terminal & GitHub
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Targeted Learning Outcomes
+- [Attention to Detail](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/04%20-%20Attention%20to%20Detail)
+- [Command Line (Unix](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/42%20-%20Command%20Line%20(Unix))
+- [Git](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/43%20-%20Git)
+- [Version Control](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/30%20-%20Version%20Control)
+- [Debugging](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/38%20-%20Debugging)
```

Comment:
@q-ode I'm curious as to how this would actually facilitate a scenario where they debug. How would that work if they are the ones who engineer the bug in the first place?